PowerPointLabs/PowerPointLabs/ELearningLab/Utility/ShapeUtility.cs

```diff
@@ -88,7 +89,15 @@ public static Shape InsertTemplatedShapeToSlide(PowerPointSlide slide, Shape tem
             float slideHeight = PowerPointPresentation.Current.SlideHeight;
 
             // templatedShape and its associated animations are duplicated
-            Shape copiedShape = templatedShape.Duplicate()[1];
+            try
+            {
+                templatedShape.Copy();               
+            }
+            catch
+            {
+                throw new Exception("Error copying shape.");
+            }
+            Shape copiedShape = slide.Shapes.Paste()[1];
```

Comment:
Should the paste code be inside the try-catch block as well? Previously copy-paste caused some problems in `ZoomToArea` and `Crop`

Reply:
Indeed... generally copy-paste might cause problems in certain machines, and especially will affect the clipboard too.

Reply:
@ChesterSng @leeyh20 Thanks for reminding! I enclosed the `paste` part to try-catch block as well.